### DISCUSSION TITLE
AcceptLanguageList setting is strictly related with the disk cache and cannot be changed

### DISCUSSION THREAD

**Discussion by bizu82** [cefsharp/CefSharp#4687] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

120.50

### Operating System

Windows 10

### Architecture

x64

### .Net Version

4.6

### Implementation

WinForms

### Reproduction Steps

- Launch the attached sample program
- Click Show DevTools from File menu
- Click Go button
- A network request to www.google.com can be observed from Network tab where Accept-Language header is set to en
- Click on Set lang to fr menu item
- Click Show DevTools from File menu
- Click Go button
- A network request to www.google.com can be observed from Network tab where Accept-Language header is set to en and not fr

[CefSample.zip](https://github.com/cefsharp/CefSharp/files/13987294/CefSample.zip)

### Expected behavior

When the user click on "Set lang to fr" a new browser instance is created with a new RequestContext having the AcceptLanguageList setting set to fr so we would expect that the Accept-Language header were set to fr but is not so. 

### Actual behavior

All Network requests have Accept-Language header set to en indipendently from the selected language.

### Regression?

We've experimented this behavior both for the last version (120.50) and on v107 too so it not seems to be a recent regression.

### Does this problem also occur in the CEF Sample Application

Not Tested

### Other information

We've observed that the language is correctly set when the RequestContext CachePath setting change. This let us think that the language is strictly related to the disk cache.

**Comment by amaitland** [comment]
> We've observed that the language is correctly set when the RequestContext CachePath setting change. This let us think that the language is strictly related to the disk cache.

You'll need to raise your issue at https://github.com/chromiumembedded/cef (post a link back here)

Read the first few paragraphs of https://github.com/cefsharp/CefSharp/wiki/General-Usage for background on the project

**Comment by amaitland** [comment]
https://github.com/chromiumembedded/cef/issues/3634

> Same cache path means same request context. Use a different (or empty) cache path if you want a different request context with different CefRequestContextSettings.accept_language_list configuration.

Behaviour is expected.
